docs/api/options.md

```diff
@@ -248,6 +248,23 @@ Please see [Other options](#other-options).
 
 Set the component instance's `$listeners` object.
 
+Example:
+
+```js
+const Component = {
+  template: '<div v-on:click="$emit(\'click\')"><slot></slot></div>'
```

Comment:
Looks great!

Could we keep this as simple and as focused as possible?

```suggestion
  template: '<button v-on:click="$emit(\'click\')"></button>'
```

(extra point for using accessible examples 😇)

Reply:
Thanks for reviewing 🙂